app/src/main/AndroidManifest.xml

```diff
@@ -15,42 +15,46 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:theme="@style/AppTheme">
+        <activity android:name=".init.SplashActivity"
+            android:theme="@style/SplashTheme">
+
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
 
         <activity
             android:name=".home.MainActivity"
             android:configChanges="orientation|screenSize"
             android:windowSoftInputMode="stateAlwaysHidden">
-
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
-                <category android:name="android.intent.category.LAUNCHER"/>
-            </intent-filter>
         </activity>
 
         <activity
-            android:label="@string/settings"
             android:name=".settings.SettingsActivity"
-            android:configChanges="orientation|screenSize">
+            android:configChanges="orientation|screenSize"
+            android:label="@string/settings">
         </activity>
 
         <activity
             android:name=".listpage.ListActivity"
+            android:configChanges="orientation|screenSize"
             android:theme="@style/AppTheme.NoActionBar"
-            android:windowSoftInputMode="stateAlwaysHidden"
-            android:configChanges="orientation|screenSize">
+            android:windowSoftInputMode="stateAlwaysHidden">
         </activity>
 
         <activity
-            android:label="@string/import_name_list"
-            android:windowSoftInputMode="adjustResize"
             android:name=".activities.ImportFileActivity"
-            android:configChanges="orientation|screenSize">
+            android:configChanges="orientation|screenSize"
+            android:label="@string/import_name_list"
+            android:windowSoftInputMode="adjustResize">
```

Comment:
Ahhh, I realize now that all of these rearrangements are for alphabetizing. That makes sense; good change :)